CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/resourcetypes/CVKCommandBuffer.java

```diff
@@ -345,24 +321,27 @@ private void bindDescriptorSetsCmd(long pipelinyLayout, long[] descriptorSets,
 //				CVKUtils.getClearColorValue(), subresourceRange);
 //	}
 //	
-//	public void drawIndexedCmd(int indexCount){
-//		
-//		vkCmdDrawIndexed(vkCommandBuffer, indexCount, 1, 0, 0, 0);
-//	}
-//	
-//	public void drawCmd(int vertexCount){
-//		
-//		vkCmdDraw(vkCommandBuffer, vertexCount, 1, 0, 0);
-//	}
+    public void DrawIndexed(int indexCount) {
+        vkCmdDrawIndexed(vkCommandBuffer, indexCount, 1, 0, 0, 0);
+    }
+
+    public void Draw(int vertexCount) {
+        
+        vkCmdDraw(vkCommandBuffer,
+                    vertexCount,    // number of verts
+                    1,              // no instancing, but we must draw at least 1 point
+                    0,              // first vert index
+                    0);             // first instance index (N/A)  
+    }
 //	
-//	public void dispatchCmd(int groupCountX, int groupCountY, int groupCountZ){
+//	public void Dispatch(int groupCountX, int groupCountY, int groupCountZ) {
```

Comment:
Are all these commented out blocks needed?

Reply:
They might still be used

Reply:
When?  The remaining renderables won't need any new paths I don't think.  Better to pull it out, it exists in git history if you need to find it.

Reply:
The renderer and hit tester have primary command buffers